en/docs/Learn/ConsumeAPI/ManageApplication/SharingApplications/sharing-applications.md

```diff
@@ -70,4 +104,23 @@ The default implementation picks the organization claim as the group ID. The org
     String[] getGroupingIdentifierList(String response);
 ```
 
-When a user signs in to the store, the latter mentioned method is called and it will return all the group IDs that correspond to the user who has logged in to the store. After users sign in to the store, they will be able to see their own applications, which they created, and also the applications that are shared with the group IDs that were returned by the `getGroupingIdentifierList` method.
+When a user signs in to the Developer Portal, the latter mentioned method is called and it will return all the group IDs that correspond to the user who has logged in to the store. After users sign in to the Developer Portal, they will be able to see their own applications, which they created, and also the applications that are shared with the group IDs that were returned by the `getGroupingIdentifierList` method.
```

Comment:
shall we change store to developer portal here?

Reply:
Fixed.